.github/workflows/deploy-production.yml

```
@@ -0,0 +1,58 @@
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy fin-in-flow to Production
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      deployments: write
+
+    environment:
```

actionlint (1.7.10)

name is missing in "environment" section

(syntax-check)